packages/bundle-size/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": ["plugin:@fluentui/eslint-plugin/node"],
+  "root": true,
+  "rules": {
+    "import/no-extraneous-dependencies": "off"
```

Comment:
Late comment but you can probably fix this by adding the following setting. 
```
  "settings": {
    "import/resolver": {
      "typescript": {
        "project": "../../tsconfig.json"
      }
    }
  },
```

Reply:
Nope, it does not work 😭 It's applied to `import/no-extraneous-dependencies` rule that has own dependencies and could be tweaked via `packageDir`:

```json
{
  "import/no-extraneous-dependencies": [
    "error",
    { "packageDir": "./some-dir/" }
  ]
}
```

I will apply this change in #18322. Thanks!